src/Compilers/Core/Portable/DocumentationCommentId.cs

```diff
@@ -301,68 +317,61 @@ private static string DecodePropertyName(string name, string language)
             return name;
         }
 
-        private class DeclarationGenerator : SymbolVisitor
+        /// <summary>
+        /// The <see langword="bool"/> type parameter indicates if we succeeded at writing out the declaration ID or not. 
```

Comment:
>type parameter indicates [](http://example.com/codeflow?start=39&length=24)

The wording feels confusing. Type parameters do not have values. Perhaps you are referring to the result returned by the `Visit` methods.

Reply:
tweaked docs.